ec/ecresource/deploymentresource/expanders_test.go

```diff
@@ -3767,15 +3716,14 @@ func Test_updateResourceToModel(t *testing.T) {
 				},
 			},
 		},
-		// The behavior of this change should be:
-		// * Resets the Elasticsearch topology: from 16g (due to unsetTopology call on DT change).
-		// * Keeps the kibana toplogy size to 2g even though the topology element has been removed (saved value persists).
-		// * Removes all other non present resources
+		// // The behavior of this change should be:
+		// // * Keeps the kibana toplogy size to 2g even though the topology element has been removed (saved value persists).
+		// // * Removes all other non present resources
```

Comment:
![yo dawg](https://i.imgflip.com/6wc3pl.jpg)

Reply:
Gotta be extra sure